implementation/jwt-build/src/test/java/io/smallrye/jwt/build/JwtEncryptTest.java

```diff
@@ -434,6 +434,23 @@ void encryptWithSecret() throws Exception {
         checkJwtClaims(claims);
     }
 
+    @Test
+    void encryptWithSecretKeyAndUseDirAlgo() throws Exception {
+        String jweCompact = Jwt.claims()
+                .claim("customClaim", "custom-value")
+                .jwe()
+                .keyId("key-enc-key-id")
+                .keyAlgorithm(KeyEncryptionAlgorithm.DIRECT)
+                .encrypt(createSecretKey());
```

Comment:
Right now it does work because the secret key is created from the same sequence, but might change. Please init a secret key var in the test and reuse it when decrypting

Reply:
I have fixed it